.github/workflows/build.yaml

```diff
@@ -157,7 +149,9 @@ jobs:
     strategy:
       matrix:
         rust:
-          - nightly-2022-07-22
+          - stable
+          - beta
+          - nightly
```

Comment:
Nightly should be a pinned version that we standardize across all the MCF repos (i.e. whatever we're going to build enclaves with)

Reply:
pinned back to 2022-07-22.
Note that the main mobile coin repo is using 2022-04-29, but I've got a hunch that will get updated prior to these crates getting brought into that repo.